src/app/loadout/plug-drawer/types.ts

```diff
@@ -1,6 +1,16 @@
 import { I18nKey } from 'app/i18next-t';
 import { PluggableInventoryItemDefinition } from 'app/inventory/item-types';
 
+/** The select behavior of the plug set. */
+export const enum PlugSelectionType {
+  /** How armor mods are selected in game, you need to manually remove ones that have been added. */
+  Multi,
```

Comment:
I've learnt my lesson with enums in typescript. Generally I avoid using them but if it makes more sense for the situation, make sure they all have the same falsey/truthy values so they can be used like truthy entities.

```suggestion
  Multi = 1,
```

Reply:
TBH I would probably turn this on in projects these days, https://typescript-eslint.io/rules/prefer-enum-initializers

Reply:
I don't think there's much danger of that here, since the selection type is not typed as nullable, but it's not a bad practice to start them at 1.